index.js

```diff
@@ -0,0 +1,73 @@
+module.exports = {
+	env: {
+		browser: true,
+		es6: true,
+	},
+	plugins: ['jsdoc', 'prettier'],
+	extends: ['airbnb-base', 'prettier', 'plugin:prettier/recommended'],
+	rules: {
+		'prettier/prettier': [
+			2,
+			{
+				useTabs: true,
+				tabWidth: 4,
```

Comment:
@nicholasio This breaks from the airbnb config right? Is there any reason to do so? Or just because that is what we had previously? 

I guess my thought process is that if we want to use airbnb-base as a starting point we should try to stick to it as much as possible unless we have a good reason.

Reply:
It does, I decided to go with tabs because that's what we're used to, most if not all of our codes are written using tabs already...